Source/WebKit/webpushd/ApplePushServiceConnection.mm

```diff
@@ -114,11 +114,8 @@ - (void)connection:(APSConnection *)connection didReceiveIncomingMessage:(APSInc
         if (!weakThis)
             return;
 
-        // FIXME: remove this once APS stops calling back on the wrong queue (rdar://88100275).
```

Comment:
What OS versions do we need this code to work on? Not sure if webpushd gets updated by the Safari installer.

Reply:
Big Sur, Monterey, Ventura, the usual suspects.

Reply:
I don't think that rdar://88100275 is fixed on all of those.

Reply:
Should be.

Reply:
This is not what I see in rdar://88100275, so please share the evidence of it having been fixed early enough.

Reply:
This file is guarded by APPLE_PUSH_SERVICE_URL_TOKEN_SUPPORT, which is enabled on macOS 13+ (Ventura) and iOS 16+. Based on rdar://88100275, this was was indeed fixed in macOS 13+  and iOS 16+ so we probably don't need the workaround anymore.

I'll let @beidson double-check my findings though.

Reply:
This was needed to fix an issue in pre-GM Ventura. From what I can tell it is fixed in GM Ventura so I'll approve it (as webpushd only runs starting in Ventura even on downlevels).

Reply:
Thank you!